python/test/dynamo/importer_basic_test.py

```diff
@@ -44,6 +44,27 @@ def basic(x):
 
         basic(torch.randn(3, 4))
 
+    def testImportDevice(self):
```

Comment:
You're accumulating a handful of these tests that do the same basic thing. When you have time, would be good to modularize them a bit so the test itself is using common code for import and mostly just has what is being tested.

Reply:
Yeah, we'll include that in the next PR to clean up the tests a bit

Reply:
should be reflected in the next PR :)